lib/spawn-git.js

```diff
@@ -8,16 +10,33 @@ var exec = require('child_process').exec
 var path = require('path')
 var os = require('os')
 
-var winGit = path.join(__dirname, '../assets/PortableGit/bin/git.exe  ')
+var winGit = path.join(__dirname, '../assets/PortableGit/bin/git.exe')
+
+type ErrorHandler = (error: ?Error, stdout: string, stderr: string) => void
 
-module.exports = function spawnGit (command, options, callback) {
-  if (typeof options === 'function') {
-    callback = options
-    options = null
+function getText(stdout: Buffer | string): string {
```

Comment:
This code feels really :shit: but I haven't really dug into whether it's correct here to be paranoid about `exec` returning a `Buffer`